### PR TITLE
[optimize](invert index) avoid redundant checks for exist.

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

File: be/src/olap/rowset/segment_v2/inverted_index_cache.cpp [hunk 1]
```diff
@@ -145,6 +145,14 @@ Status InvertedIndexSearcherCache::get_index_searcher(
         return Status::OK();
     }
 
+    // check index file existence
```
Comment (airborne12) [inline be/src/olap/rowset/segment_v2/inverted_index_cache.cpp:148]: line 162 below also checks index exists

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**